src/org/openstreetmap/josm/plugins/osmobjinfo/OSMObjInfoActions.java

```diff
@@ -64,26 +72,75 @@ public static void openinBrowserChangesetMap(String idChangeset) {
+        if (osmObjInfo == null || osmObjInfo.isEmpty()) return;
+        String linkobjid = "";
+        StringBuilder builder = new StringBuilder();
+        for (AllOsmObjInfo object : osmObjInfo) {
+            builder.append("https://www.openstreetmap.org/")
+                .append(object.typeObj).append("/").append(object.idObject);
+            if (osmObjInfo.indexOf(object) < osmObjInfo.size() - 2) {
+                builder.append(", ");
+            }
+            if (osmObjInfo.indexOf(object) == osmObjInfo.size() - 2) {
+                if (osmObjInfo.size() == 2) {
+                    builder.append(" ");
+                } else {
+                    builder.append(", ");
+                }
+                builder.append(tr("and"));
+                builder.append(" ");
+            }
         }
+        linkobjid = builder.toString();
+        StringSelection selection = new StringSelection(linkobjid);
+        Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
+        clipboard.setContents(selection, selection);
+        new Notification(tr(COPY, linkobjid)).setIcon(JOptionPane.INFORMATION_MESSAGE).setDuration(Notification.TIME_SHORT).show();
+    }
+    
+    public static void copyRemoteControl(List<AllOsmObjInfo> osmObjInfo) {
+    	if (osmObjInfo == null || osmObjInfo.isEmpty()) return;
+    	StringBuilder builder = new StringBuilder();
+        builder.append("http://127.0.0.1:8111/load_and_zoom?");
+        if (!osmObjInfo.isEmpty())
+            builder.append("select=");
+    	for (int i = 0; i < osmObjInfo.size(); i++) {
+    		AllOsmObjInfo object = osmObjInfo.get(i);
+    		builder.append(object.typeObj);
+    		builder.append(object.idObject);
+    		if (i < osmObjInfo.size() - 1)
+    			builder.append(',');
+    	}
+    	Bounds viewPortBound = MainApplication.getMap().mapView.getLatLonBounds(MainApplication.getMap().mapView.getVisibleRect());
+    	builder.append("&left=").append(viewPortBound.getMinLon())
+    	    .append("&right=").append(viewPortBound.getMaxLon())
+    	    .append("&top=").append(viewPortBound.getMaxLat())
+    	    .append("&bottom=").append(viewPortBound.getMinLat());
+    	String remoteControl = builder.toString();
+    	StringSelection selection = new StringSelection(remoteControl);
+        Clipboard clipboard = Toolkit.getDefaultToolkit().getSystemClipboard();
+        clipboard.setContents(selection, selection);
+        new Notification(tr(COPY, remoteControl)).setIcon(JOptionPane.INFORMATION_MESSAGE).setDuration(Notification.TIME_SHORT).show();
+
     }
 
-    public static void openinBrowserIdobj(String typeObj, String idobj) {
-        if (typeObj != null && !idobj.isEmpty()) {
-            openInBrowser("https://www.openstreetmap.org/" + typeObj + "/" + idobj);
+    public static void openinBrowserIdobj(List<AllOsmObjInfo> osmObjInfo) {
+        if (osmObjInfo.size() > 5 || osmObjInfo.isEmpty()) return;
```

Comment:
why 5 ?

Reply:
I picked it arbitrarily. I should probably change it to be a config option.